app/views/deals/_deal.html.erb

```diff
@@ -0,0 +1,25 @@
+<div id="<%= dom_id deal %>">
+  <u><%= link_to "#{deal.title}", admin_deal_path(deal) %></u>
+  </br>
+
+  <div>
+    <% deal.deal_images.each do |deal_image| %>
+      <%= image_tag(url_for(deal_image.image)) %>
+    <% end %>
+  </div>
+
+  <div>
+    <strong>Description </strong>
+    <%= deal.description %>
+  </div>
+
+  <div>
+    <strong>Original Price</strong>
+    <s><%= price_in_decimal(deal.price_in_cents).to_fs(:currency, format: '%u %n') %></s>
```

Comment:
let's create a method called price on deal model and use it everywhere you are using price_in_decimal